cypress.config.js

```diff
@@ -2,6 +2,7 @@ const { defineConfig } = require('cypress');
 
 module.exports = defineConfig({
   e2e: {
+    baseUrl: 'https://demoqa.com',
```

Comment:
The 'baseUrl' should be in the root level of the configuration object, not inside the 'e2e' object.